Userland/Libraries/LibCore/ArgsParser.cpp

```diff
@@ -500,6 +500,25 @@ void ArgsParser::add_option(Optional<size_t>& value, const char* help_string, co
     add_option(move(option));
 }
 
+void ArgsParser::add_option(Vector<size_t>& values, char const* help_string, char const* long_name, char short_name, char const* value_name, char separator)
+{
+    Option option {
+        true,
+        help_string,
+        long_name,
+        short_name,
+        value_name,
+        [&values, separator](char const* s) {
+            auto separated_values = String(s).split(separator, false);
+            for (auto value : separated_values)
```

Comment:
Just thinking, would it be better to use const references here?

Reply:
Since these are `String`s, it would be. But we can avoid both `String` and the extra `Vector` allocation here. And we should not assume that `convert_to_uint` will succeed. Something like (untested):

```c++
bool parsed_all_values = true;

StringView { s }.for_each_split_view(separator, false, [&](auto value) {
    if (auto maybe_value = AK::StringUtils::convert_to_uint<size_t>(value); maybe_value.has_value())
        values.append(*maybe_value);
    else
        parsed_all_values= false;
});

return parsed_all_values;
```

Reply:
Thanks! That's a much better solution